### ISSUE TITLE
Install package in edit mode success but nothing shows

### ISSUE THREAD

**Issue by unkcpz** [aiidalab/widget-bandsplot#53]
I install the package with `pip install -e .` The installation finished without issue but the widget is not working. However, when I install it without edit mode, the installation fails and asks for `yarn`. I install yarn and the installation was finished but still, the widget is not shown.

**Comment by dou-du** [comment]
I do not have problem for local installation. Do you use ipywidgets 8.x? Currently, it is not supported ipywidgets 8.x yet.

**Comment by unkcpz** [comment]
No, I use ipywidgets 7.7.2, and the ipywidgets version is pinned in the `setup.cfg`. But I restart the browser the bands show again. I'll close this issue for the moment but I think it needs some documentation to install the widget in edit mode and ask the user to restart the browser. @superstar54 also test it in the QeApp, and install the widget in edit mode has the problem. @superstar54, can you restart the browser or restart the container to see if the issue is resolved?
